### ISSUE TITLE
Migrate to UXP

### ISSUE THREAD

**Issue by bossanovaorca** [Swirt/typertools-src#53]
hello, are there any plans to migrate this plugin to UXP? I believe this will also resolve any compatibility issues people are having with Apple Silicon/macOS #21 #38

**Comment by Swirt** [comment]
Yes, I'm thinking about it. But I don't know when I'll get around to it.

**Comment by korafu** [comment]
@Swirt what would be the best way to help with this effort? I haven't built a UXP plugin before but I am an experienced React developer who is interested in contributing to this

so far I've scaffolded a UXP plugin but I need to familiarize myself with the codebase to start porting features over. I can share my (basic) feature branch if you're interested in taking a look

**Comment by Swirt** [comment]
> what would be the best way to help with this effort? 
It's hard to say, since I am also a React developer and am not yet familiar with UXP. Besides, the React component is the easiest part of the expansion. And it is likely to remain mostly unchanged. 
It's more important to figure out how the use of js injections into the Photoshop engine has changed (now it's in the [app_src/host.js](https://github.com/Swirt/typertools-src/blob/master/app_src/host.js) file). But there is very little documentation on this topic.
